packages/catalog-server/src/lib/firestore/firestore-repository.ts

```diff
@@ -4,13 +4,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
BSD or Apache? Same for all files, I just noticed the headers are not consistent with the package.json

Reply:
Everything should be Apache now